hail/python/test/hail/genetics/test_reference_genome.py

```diff
@@ -1,174 +1,186 @@
-import unittest
-
 import hail as hl
 from hail.genetics import *
 from ..helpers import *
 from hail.utils import FatalError
 
 
-class Tests(unittest.TestCase):
```

Comment:
Sorry for the diff. I'm on a mission to excise unittest.

Reply:
Why?

Reply:
I dislike the extra four spaces in front of each test case. I'd rather isolate tests by file than class.